### ISSUE TITLE
Overwriting matplotlib `PolarAxes` when registering projections raises an `AttributeError`

### ISSUE THREAD

**Issue by kikocorreoso** [lukelbd/proplot#141]
### Description

When new projections are registered there is a conflict with already registered matplotlib projections [here](https://github.com/lukelbd/proplot/blob/master/proplot/axes.py#L4097).

### Steps to reproduce

For instance, the following works:

```python
import matplotlib.pyplot as plt

fig, ax = plt.subplots(
    ncols=1, 
    nrows=1, 
    figsize=(10, 10), 
    subplot_kw={"projection": "polar"}
)
```

The following crashes:

```python
import matplotlib.pyplot as plt
import proplot as plot # just importing proplot register new projections

fig, ax = plt.subplots(
    ncols=1, 
    nrows=1, 
    figsize=(10, 10), 
    subplot_kw={"projection": "polar"}
)
```
with error:
```python
AttributeError: 'SubplotSpec' object has no attribute 'get_active_rows_columns'
```

### Equivalent steps in matplotlib

Shown in previous section.

### Proplot version

`0.5.0`

Installed using conda.

Matplotlib version is `3.2.0`

**Comment by lukelbd** [comment]
Thanks for the report, that's a major issue. I'll rename proplot's `PolarAxes` to something like `'polar_proplot'` and have `proplot.subplots` translate user input `proj='polar'` to the new name.

**Comment by lukelbd** [comment]
This was fixed in #149  (4a6a0e34).
